### ISSUE TITLE
missing definition of 'list'

### ISSUE THREAD

**Issue by jelmd** [docker/cli#2138]
`docker run --help` etc. mentions several times  __list__ as an option argument but does not define, what a list actually is. E.g.:
```
      --dns list                       Set custom DNS servers
      --dns-option list                Set DNS options
      --dns-search list                Set custom DNS search domains
```

I've tried e.g. --dns '1.2.3.4,5.6.7.8' but this does not work and results into
```
invalid argument "1.2.3.4,5.6.7.8" for "--dns" flag: 1.2.3.4,5.6.7.8 is not an ip address
See 'docker run --help'.
```

Also the man page is wrong: It shows e.g.
```
--dns=[]  Set custom DNS servers
```

__[]__ denotes optional arguments in man pages. Since it does not contain any argument, it is actually the same as `--dns` (does not allow any argument).

So it should be documented/defined, what a list is/what docker actually expects as argument.

**Comment by nabeel-shakeel** [comment]
@jelmd : yeah, on my Desktop Docker client too, the `docker help` shows the same `--dns list`.

But it has updated in the run [manual](docs/reference/commandline/run.md), that its not **list** its a **single value** which this argument excepts.
like this `--dns 8.8.8.8` and to specify multiple DNS servers, use multiple --dns flags.

Is this change already rolled out in new update of Docker ? If not, then i would like to contribute to make this change.
